okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -344,6 +347,73 @@ static Status toGrpcStatus(ErrorCode code) {
     return ERROR_CODE_TO_STATUS.get(code);
   }
 
+  private Socket upgradeToTls(Socket socket, SSLSocketFactory sslSocketFactory) throws IOException {
```

Comment:
I'm not sure, I don't know much about the Ciphers, what I'm referring to is actually the Netty code in https://github.com/netty/netty/blob/8eddeb1ddd958c872a59cc111cf036ede92c4882/handler/src/main/java/io/netty/handler/ssl/JdkSslContext.java, I noticed it's a bit different than what OkHttp used, since we only has Netty transport on the server, I'm making our config more align with Netty.